src/SFA.DAS.Roatp.CourseManagement.Web/Infrastructure/RouteNames.cs

```diff
@@ -39,8 +40,8 @@ public static class RouteNames
 
         public const string GetAddProviderLocationDetails = "GetAddProviderLocationDetails";
         public const string PostAddProviderLocationDetails = "PostAddProviderLocationDetails";
-
         public const string GetUpdateProviderLocationDetails = "GetUpdateProviderLocationDetails";
         public const string PostUpdateProviderLocationDetails = "PostUpdateProviderLocationDetails";
+        public const string ProviderDescription = "provider-description";
```

Comment:
This should follow the convention `GetProviderDescription`

Reply:
fixed